distribution/docs/src/main/jdocs/content/_plugins/resource-usage-plugin-contents.adoc

```diff
@@ -0,0 +1,24 @@
+:type: plugin
+:status: published
+:title: Resource Usage Plugin
+:link: _resource_usage_plugin
+:plugintypes: preresource,postquery,postresource
+:summary: Monitors and limits system data usage.
+
+The Resource Usage Plugin monitors and limits data usage, and enables cancelling long-running queries.
```

Comment:
To my knowledge there is no way to cancel a resource download.  This appears to describing the functionality that exists in the query monitor.  I think it would be more accurate to move `and enables cancelling long-running queries.` to that plugin instead
https://github.com/codice/ddf/tree/master/catalog/resourcemanagement/resourcemanagement-querymonitor-plugin